.github/workflows/ci.yaml

```diff
@@ -0,0 +1,33 @@
+name: continuous integration
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  pull_request:
```

Comment:
I see from your commits that you added this to test the workflow. Is it necessary to keep this event in? Not sure we want to be rebuilding/pushing the Docker image at the PR stage.

Reply:
Yea just wanted to test the workflow, I planned on deleting it before merging.